### ISSUE TITLE
Implement GCS for `pachctl deploy storage ...`

### ISSUE THREAD

**Issue by dwhitena** [pachyderm/pachyderm#2578]
`pachctl deploy storage ...` allows users to pull in object store data into Pachyderm running in one cloud from another cloud. This is currently implemented for aws and Azure, but not GCP (i.e., GCS):

```
joeyzwicker$ pachctl deploy storage azure
Usage: pachctl deploy storage azure <account name> <account key>
JosephZckersMBP:~ joeyzwicker$ pachctl deploy storage google
deploying credentials for GCS storage is not currently supported
JosephZckersMBP:~ joeyzwicker$ pachctl deploy storage aws
Usage: pachctl deploy storage aws <region> <id> <secret> <token>
```

**Comment by JoeyZwicker** [comment]
@jdoliner or @sjezewski -- the framework for this is all there, but GCS just hasn't been implemented yet. Should be pretty easy. 

Not sure if this is relevant, but the user also said, "I get that gcs is a pain because it *must* be an environment variable to a json file"

**Comment by jdoliner** [comment]
So it turns out that the reason this wasn't implemented initially is that our current GCS library only supports getting credentials through GCE node's credentials, you can't encode the creds as a secret and add them to, for example, an aws node. This shouldn't be too hard to fix, there's a new library that we can use that should do what we want. Still it'll require migrating to a new library so it's not totally trivial work.

**Comment by jdoliner** [comment]
Fixed in #2610
